src/telegram.ts

```diff
@@ -129,10 +131,11 @@ export class Telegram extends ApiClient {
    */
   sendMessage(
     chatId: number | string,
-    text: string,
+    text: string | FmtString,
     extra?: tt.ExtraReplyMessage
   ) {
-    return this.callApi('sendMessage', { chat_id: chatId, text, ...extra })
+    const t = typeof text === 'string' ? new FmtString(text) : text
+    return this.callApi('sendMessage', { chat_id: chatId, ...extra, ...t })
```

Comment:
This makes it impossible to manually pass custom `parse_mode`.

```suggestion
    const t = FmtString.normalise(text)
    return this.callApi('sendMessage', { chat_id: chatId, ...t, ...extra })
```

`editMessage*` should be updated as well.

Reply:
> ...impossible to manually pass custom `parse_mode`

Fixed in 18a88154efb3b3fdd84cc988c2382c7724d771b7

> `editMessage*` ...

Fixed in a0ece2e9b029ba71f786cfa0c8de7d24ded0b48b